kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSidePublicTab.js

```diff
@@ -658,6 +658,7 @@ define ([
             var self = this;
             var type = object.type.split('.')[1].split('-')[0];
             var copyText = ' Add';
+            var baseClass = 'kb-object-row';
```

Comment:
I reworked this function to remove the inline css declarations and a couple of extra unneeded divs. The css is in the `_objectRow.scss` partial.